pipeline/05-finalize.R

```diff
@@ -12,10 +12,12 @@ suppressPackageStartupMessages({
   library(aws.ec2metadata)
   library(ccao)
   library(dplyr)
+  library(glue)
   library(here)
   library(lubridate)
   library(paws.application.integration)
   library(purrr)
+  library(quarto)
```

Comment:
Per my note above, one way we could make this smoother in the case where the user doesn't choose to install `reporting` dependencies would be to check if this library is installed or not and skip the generation/upload steps below if so.

Reply:
Per my comment above, `finalize` should now be runnable by outside users, but we've also added a `tryCatch` block around report generation that should fail more gracefully if report dependencies are not installed.